File: Gerador.js
const chars = {
    "letters": "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZzÇç",
    "numbers": "0123456789",
    "symbols": "!@#$%&*()-_+=^{}[];:?",
};
let seed;
let password = "";
function randomInteger(min, max) {
    return Math.floor(Math.random() * (max - min + 1)) + min;
};
function Generate(length) {
    for (let i = 0; i < length; i++) {
        seed = randomInteger(1, 3);
        switch (seed) {
            case 1:
                seed = randomInteger(0, chars.letters.length - 1);
                password += chars.letters[seed];
                break;
            case 2:
                seed = randomInteger(0, chars.numbers.length - 1);
                password += chars.numbers[seed];
                break;
            case 3:
                seed = randomInteger(0, chars.symbols.length - 1);
                password += chars.symbols[seed];
                break;
        };
    };
    console.log(`Your password is "${password}"`);
    return password = "";
};

Generate(15)

File: Gerador.py
import random 
chars = {
    "letters": "AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZzÇç",
    "numbers": "0123456789",
    "symbols": "!@#$%&*()-_+=^{}[];:?",
};
seed = 0

def Gerador(length):
    password = ""
    for i in range(length):
        seed = random.randint(1,3)
        match seed:
            case 1:
                seed = random.randint(0,chars["letters"].__len__() - 1)
                password += chars["letters"][seed]
            case 2:
                seed = random.randint(0,chars["numbers"].__len__() - 1)
                password += chars["numbers"][seed]
            case 3:
                seed = random.randint(0,chars["symbols"].__len__() - 1)
                password += chars["symbols"][seed]
    print("Your password is", password)
    password = ""

Gerador(15)
